tests/gem/test_api.py

```diff
@@ -44,22 +44,33 @@ def setUp(self):
         self._protocol.connection_made(self._transport)
 
     async def testApiCallWithoutResponse(self):
-        call = ApiCall(lambda _: "REQUEST", None, None, None)
-
-        await self.assertCall(call, "REQUEST", None, None, None, None)
+        await self.assertCall(
+            ApiCall(lambda _: "REQUEST", None, None, None),
```

Comment:
This is annoying, but without doing it, pyright was very, very confused.  Same with the next two tests in this file.